tests/cpa-config.yml

```diff
@@ -0,0 +1,20 @@
+cratedb_endpoints:
+- host: "localhost"         # Host to connect to (default: "localhost").
+  port: 5432                # Port to connect to (default: 5432).
+  user: "crate"             # Username to use (default: "crate")
+  password: ""              # Password to use (default: "").
+  schema: "testdrive"       # Schema to use (default: "").
```

Comment:
@surister: That is the configuration slot how to configure the program to also use that schema name instead of the default `doc`.